src/components/Datepicker/Datepicker.vue

```diff
@@ -375,6 +427,25 @@ export default {
       if (this.internalValue === 'Invalid Date') this.internalValue = ''
     },
 
+    isDateDisabled() {
+      let valid = false
+      if (this.disabledPast && dayjs().isAfter(this.internalValue, 'day')) {
+        valid = true
+      } else if (
+        this.minDate &&
+        dayjs(this.internalValue).isSameOrBefore(this.minDate, 'day')
+      ) {
+        valid = true
+      } else if (
+        this.maxDate &&
+        dayjs(this.internalValue).isSameOrAfter(this.maxDate, 'day')
+      ) {
+        valid = true
+      }
+
+      return valid
+    },
```

Comment:
Maybe here you can just return true or false inside each if statement, like:

```js
isDateDisabled() {
  if (this.disabledPast && dayjs().isAfter(this.internalValue, 'day')) {
    return true
  }
  if (
    this.minDate &&
    dayjs(this.internalValue).isSameOrBefore(this.minDate, 'day')
  ) {
    return true
  }
  if (
    this.maxDate &&
    dayjs(this.internalValue).isSameOrAfter(this.maxDate, 'day')
  ) {
    return true
  }
  return false
}
```

Reply:
I did it with the variable to reduce the complexity of the code, with the 'let' we only have 1 return instead of 4.